sdk/ml/azure-ai-ml/azure/ai/ml/entities/_assets/_artifacts/feature_set.py

```diff
@@ -52,6 +52,7 @@ class FeatureSet(Artifact):
     :raises ValidationException: Raised if stage is specified and is not valid.
 
     .. admonition:: Example:
+
         .. literalinclude:: ../../../../../../samples/ml_samples_featurestore.py
```

Comment:
Hi Alan, could you please update any of the samples sections you see to have this path prefix

```
../samples/<file name>
```

instead of 

```
./../../../../../samples/<file name>
```

We had a change last week after discussion with the team that builds the Sphinx docs and the examples will only be rendered if they're using this file pattern.

Reply:
https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/ml/azure-ai-ml/samples/README.md

Reply:
Fixed paths